CDCgov/prime-reportstream
# 14929 - Archived File Validator

This PR ...

Removes the links and archive the file validator tool

Test Steps:
1. Log into RS
2. Check Manage your Connection Page and the Admin dropdown for validation
3. /file-handler/validate will redirect you to the API -> Getting Started page

## Changes


## Checklist

### Testing


## Linked Issues
- Fixes #14929
